packages/data-models/flowTypes.ts

```diff
@@ -373,7 +373,7 @@ export namespace FlowTypes {
       | "track_event"
       | "process_template"
       | "google_auth"
-      | "set_highlighted_task";
+      | "set_highlighted_task_group";
```

Comment:
nit(non-blocking)
would be better to be prefix with `task` so easier to couple with functionality, e.g. `task_group_set_highlighted`